server/views/pages/overview/partials/supportNeedsTabContents.test.ts

```diff
@@ -22,6 +23,19 @@ describe('Support Needs tab view', () => {
     compiledTemplate = nunjucks.compile(template.toString(), njkEnv)
   })
 
+  it('should render content', () => {
+    // Given
+    const supportNeeds = aValidPrisonerSupportNeeds()
+    viewContext = { prisonerSummary, tab: 'support-needs', supportNeeds }
+
+    // When
+    const $ = cheerio.load(compiledTemplate.render(viewContext))
+
+    // Then
+    expect($('#health-and-support-needs-summary-card')).not.toBeUndefined()
+    expect($('#neurodiversity-summary-card')).not.toBeUndefined()
```

Comment:
A bit basic and nothing more than the cypress test is asserting but better than nothing (and we get quick feedback if something is broken, compared to the int tests.

I could drill into the data and assert specific values are shown if we like? A bit fiddly but possible.